docs/topics/native/apple-framework.md

```diff
@@ -400,17 +401,17 @@ if (ret != nil) {
 }
 ``` 
 
-There are some small differences between the original Kotlin code and its Swift version. In Kotlin, any `object` has 
+There are some small differences between the original Kotlin code and its Swift version. In Kotlin, any object has 
```

Comment:
Now, after removing backticks around "object", the statement "In Kotlin, any object has only one instance" is confusing.
Like, in `val x = Foo()`, there is a new object created. What does this statement mean for that object?

I think using "object declaration" instead of "object" would be less confusing.

Reply:
Right, thank you for noticing!